### ISSUE TITLE
New Folder Button on Filepicker Dialog (Type Choose)

### ISSUE THREAD

**Issue by TessyPowder** [nextcloud/server#19438]
**Is your feature request related to a problem? Please describe.**
I am currently working on a PR to a nextcloud app called flowupload (https://github.com/e-alfred/flowupload/pull/73).
In my PR you can chose the upload directory with a filepicker Dialog and it would be really useful to have a button to create a new folder.

This button exists on different filepicker Types (like OC.dialogs.FILEPICKER_TYPE_COPY):
![image](https://user-images.githubusercontent.com/28999431/74350213-ac199b00-4db5-11ea-8e3f-eb86bdb259d1.png)

... but not on OC.dialogs.FILEPICKER_TYPE_CHOOSE:
![image](https://user-images.githubusercontent.com/28999431/74350323-de2afd00-4db5-11ea-9619-3b8b6a86efd0.png)
... it doesn't exist

**Describe the solution you'd like**
A button like on FILEPICKER_TYPE_COPY would be nice.

**Additional context**
This is the code i open the dialog with:
```javascript
OC.dialogs.filepicker("Select a new Upload Folder", function(path) { ... }, false, 'httpd/unix-directory', true, OC.dialogs.FILEPICKER_TYPE_CHOOSE);
```

**Comment by kesselb** [comment]
https://github.com/nextcloud/server/blob/aa790a3961e49fcb996190dc3e3820dc6fce4011/core/src/OC/dialogs.js#L301-L304

It seems the button is already there just hidden. I would try to make it visible. Probably a bit cleaner than making the button visible is to use `FILEPICKER_TYPE_CUSTOM` (not sure if available for Nextcloud 17) and push/trigger the choose action manually.

**Comment by kesselb** [comment]
cc @nextcloud/javascript opinions? I think it should be possible by using FILEPICKER_TYPE_CUSTOM but haven't tested it.

**Comment by TessyPowder** [comment]
Is there any reason the button is hidden ?

**Comment by kesselb** [comment]
Might be fixed by https://github.com/nextcloud/server/pull/19539

**Comment by TessyPowder** [comment]
> Might be fixed by #19539

This change is in the branch stable19 and it does fix my problem 👍
